code/__HELPERS/logging/admin.dm

```diff
@@ -3,31 +3,26 @@
 /// General logging for admin actions
 /proc/log_admin(text)
 	GLOB.admin_log.Add(text)
-	if (CONFIG_GET(flag/log_admin))
-		WRITE_LOG(GLOB.world_game_log, "ADMIN: [text]")
+	logger.Log(LOG_CATEGORY_ADMIN, text)
 
 /// Logging for admin actions on or with circuits
 /proc/log_admin_circuit(text)
 	GLOB.admin_log.Add(text)
-	if(CONFIG_GET(flag/log_admin))
-		WRITE_LOG(GLOB.world_game_log, "ADMIN: CIRCUIT: [text]")
+	logger.Log(LOG_CATEGORY_ADMIN_CIRCUIT, text)
 
 /// General logging for admin actions
 /proc/log_admin_private(text)
 	GLOB.admin_log.Add(text)
-	if (CONFIG_GET(flag/log_admin))
-		WRITE_LOG(GLOB.world_game_log, "ADMINPRIVATE: [text]")
+	logger.Log(LOG_CATEGORY_ADMIN_PRIVATE, text)
 
 /// Logging for AdminSay (ASAY) messages
 /proc/log_adminsay(text)
 	GLOB.admin_log.Add(text)
-	if (CONFIG_GET(flag/log_adminchat))
-		WRITE_LOG(GLOB.world_game_log, "ADMINPRIVATE: ASAY: [text]")
+	logger.Log(LOG_CATEGORY_ADMIN_PRIVATE_ASAY, text)
 
 /// Logging for DeachatSay (DSAY) messages
 /proc/log_dsay(text)
-	if (CONFIG_GET(flag/log_adminchat))
-		WRITE_LOG(GLOB.world_game_log, "ADMIN: DSAY: [text]")
+	logger.Log(LOG_CATEGORY_ADMIN_DSAY, text)
 
```

Comment:
for now can we please keep logs that go to the game.log file going to game.json logfile.

at least until full tooling and viewing merged files is properly set up.

an alternative could include sending it to both.